### ISSUE TITLE
Add Landsat-8 notebook from pangeo-tutorial

### ISSUE THREAD

**Issue by salvis2** [pangeo-gallery/pangeo-gallery#22]
/add-submodule pangeo-data/landsat-8-tutorial-gallery

**Comment by salvis2** [comment]
I have some errors in the GitHub actions on my end, working on those.

**Comment by salvis2** [comment]
So I've been trying to build the finished notebook and I'm getting an error I don't know how to diagnose: https://github.com/pangeo-data/landsat-8-tutorial-gallery/runs/754832034?check_suite_focus=true

Essentially, that boils down to: 

```
CellExecutionError: An error occurred while executing the following cell:
------------------
import geopandas as gpd
gfa = gpd.read_file('wa-bbox.json')
gfa.plot()
------------------

---------------------------------------------------------------------------
CPLE_OpenFailedError                      Traceback (most recent call last)
fiona/_shim.pyx in fiona._shim.gdal_open_vector()

fiona/_err.pyx in fiona._err.exc_wrap_pointer()

CPLE_OpenFailedError: wa-bbox.json: No such file or directory
```

This error is unexpected because the file `wa-bbox.json` is the repo with the notebook: https://github.com/pangeo-data/landsat-8-tutorial-gallery 

What should I do?

**Comment by rabernat** [comment]
HI @salvis2 - did you find a workaround for this?

**Comment by salvis2** [comment]
I did! Another issue has popped up with a different package's API and I'm waiting on a release for that package, so it might be a bit before this tutorial can be gallery'd.

**Comment by salvis2** [comment]
@rabernat @andersy005 Can one of you re-run the job? https://github.com/pangeo-gallery/pangeo-gallery/runs/754803082?check_suite_focus=true I am not authorized.

I got the issue above sorted out and the build seemed to work except for dispatching to this repo: https://github.com/pangeo-data/landsat-8-tutorial-gallery/runs/824631286

**Comment by rabernat** [comment]
Thanks for your persistence on this @salvis2! 😁 

I triggered the job to re-run. I also added you to the gallery-admin team so that you have admin access to this repo. Please feel free to make any changes you see fit to make things work better!

**Comment by salvis2** [comment]
Thanks @rabernat ! I'll use this power wisely and will at least bug you less about the small stuff. :)

**Comment by salvis2** [comment]
Looks like the GitHub Action was able to partially build the branch but is having problems updating it. I'm going to delete the branch and try a fresh Action.

**Comment by salvis2** [comment]
Action worked in #26 . PR in #27 .

**Comment by salvis2** [comment]
Okay, the Landsat 8 notebook is up and running. Some formatting bugs, but I can look at those later.
